### PR TITLE
fix(material-experimental/button): minor comment changes

### PR DESCRIPTION
**Pull request by andrewseguin**
Just a few minor places where comments need to be added in

### CODE REVIEW

**Comment by andrewseguin** [comment]
Updated with more thorough jsdocs for the classes

File: src/material-experimental/mdc-button/button.ts [hunk 1]
```diff
@@ -27,7 +27,15 @@ import {
   MatButtonBase
 } from './button-base';
 
-
+/**
+ * Material Design button component. Users interact with a button to perform an action.
+ * See https://material.io/components/buttons
+ *
+ * The `MatButton` class applies to native button elements and captures the appearances for
+ * "text button", "outlined button", and "contained button" per the Material Design
```
Comment (mmalerba) [inline src/material-experimental/mdc-button/button.ts]: Does it do outlined buttons? the README still mentions "Consider supporting button[mat-outlined-button]"

File: src/material-experimental/mdc-button/button.ts [hunk 2]
```diff
@@ -50,6 +58,16 @@ export class MatButton extends MatButtonBase {
   }
 }
 
+/**
+ * Material Design button component for anchor elements. Anchor elements are used to provide
+ * links for the user to navigate across different routes or pages.
+ * See https://material.io/components/buttons
+ *
+ * The `MatAnchor` class applies to native anchor elements and captures the appearances for
+ * "text button", "outlined button", and "contained button" per the Material Design
```
Comment (mmalerba) [inline src/material-experimental/mdc-button/button.ts]: ditto about outline buttons

File: src/material-experimental/mdc-button/button.ts (hunk 1 above)
Comment (jelbourn) [reply src/material-experimental/mdc-button/button.ts]: We support outlined buttons, but they're _called_ `mat-stroked-button`. That TODO was about whether we want to add `mat-outlined-button` as an alias.

**Pull request merged**